tests/datatypes/test_set.py

```diff
@@ -62,6 +62,34 @@ def test_iter(self):
             """)
 
 
+    def test__ior__(self):
+        self.assertCodeExecution("""
+        a = {'a', 'b', 'c'}
+        b = {'a', 'c', 'd'}
+        a.__ior__(b)
+        """)
+
+    def test__iand__(self):
+        self.assertCodeExecution("""
+        a = {'a', 'b', 'c'}
+        b = {'a', 'c', 'd'}
+        a.__iand__(b)
+        """)
+
+
+    def test_union(self):
+        self.assertCodeExecution("""
+        a = {'a', 'b', 'c'}
+        b = {'a', 'c', 'd'}
+        a.union(b)        
+        """)
+
+    def test_intersection(self):
+        self.assertCodeExecution("""
+        a = {'a', 'b', 'c'}
+        b = {'a', 'c', 'd'}
+        a.intersection(b)
```

Comment:
As with `union`, a print statement (or something else that generates output) is required.